### ISSUE TITLE
PM Review #2: User Auth

### ISSUE THREAD

**Issue by andrewlangston** [dkdan10/SquadUp#2]
Hey Daniel,

Great work so far on your fullstack. Below are some notes on your user auth feature. I'll open a separate issue for groups/joining groups once that functionality is complete.

1. Some minor CSS tweaks to be made:
- [ ] See if you can hunt down a logo generator to stylize "Squad Up" in your nav - it should be more eye-catching than plain text
- [ ] There are some slight spacing & sizing differences between your app and Meetup. For example: your nav bar has greater height, your user's icon (in the nav bar) is smaller, etc. Pull your app up side-by-side with Meetup to compare. Aim for pixel perfect recreation!

2. Session error handling needs to better approximate Meetup:
- [ ] Rather than rendering an unordered list of errors served up by the backend, it looks like Meetup renders errors inline (that is, adjacent to the input field to which they are related), as well as a large notification at the top of the page if trying to log in with erroneous info. You should do the same.

3. Add a footer to your session form pages:
- [ ] These can be valuable real estate for linking your LinkedIn, Github, and AngelList profile. Feel free to get creative with how you fill this space - but do make sure to fill it.

Please address these comments by Monday, and tackle as much of events/RSVPs/calendar as you can. It'd be awesome to get ahead so that you have time to reach bonus features like messaging come next week. Let me know if you have questions!
